Bottom: 021491e87d7a4a576c6d2f7f97025d6ef1155a41
Top:    c89745d7513e3361c5fa8f591c0d916333fd110f
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-01-11 14:36:38 +0000

Refresh of w32-add-recipe-for-freetype

---

diff --git a/w32/recipes/freetype.recipe b/w32/recipes/freetype.recipe
new file mode 100644
index 0000000..1806541
--- /dev/null
+++ b/w32/recipes/freetype.recipe
@@ -0,0 +1,4 @@
+name=freetype
+version=2.4.4
+release=1
+origin=sf
